.mailmap

```diff
@@ -323,12 +328,14 @@ Sam Mikes <smikes@cubane.com>
 Sam P Gallagher-Bishop <samgallagherb@gmail.com> <SPGB@users.noreply.github.com>
 Sam Shull <brickysam26@gmail.com> <brickysam26@samuel-shulls-computer.local>
 Sam Shull <brickysam26@gmail.com> <sshull@squaremouth.com>
-Sambasiva Suda <sambasivarao@gmail.com>
 Sam Roberts <vieuxtech@gmail.com> <sam@strongloop.com>
+Samantha Sample <ssample812@gmail.com> = <=>
```

Comment:
```suggestion
Samantha Sample <ssample812@gmail.com>
```

Unexpected characters?

Reply:
@legendecas That’s literally the `Author:` field of e57bf473517f12eb169e960aa80ffd5dcd5994d5. No idea how that happened but it did happen.